Cargo.toml

```diff
@@ -37,7 +37,7 @@ serde_derive = "1.0"
 winapi = { version = "0.3", features = ["sspi", "rpcdce", "impl-default", "timezoneapi", "wincrypt"] }
 url = "2.2.2"
 reqwest = { version = "0.11", features = ["blocking", "rustls-tls", "rustls-tls-native-roots"], optional = true, default-features = false }
-picky-krb = "0.3.0"
+picky-krb = { git = "https://github.com/Devolutions/picky-rs.git", rev = "8a05c988832722af2628d2270c46cb6b47c1c41d" }
```

Comment:
I will update the `picky-krb` dependency once the version is bumped

Reply:
I just published now :slightly_smiling_face:

Reply:
Thanks! 
I pushed a commit that sets `picky-krb` to 0.3.1